README.md

```diff
@@ -7,9 +7,9 @@ Puppet Catalog Diff Viewer
 [![By Camptocamp](https://img.shields.io/badge/by-camptocamp-fb7047.svg)](http://www.camptocamp.com)
 
 
-A viewer for json reports produced by [the puppet-catalog-diff tool](https://github.com/camptocamp/puppet-catalog-diff)
+A viewer for json reports produced by [the puppet-catalog-diff tool](https://github.com/voxpupuli/puppet-catalog-diff)
 
-The interface can be tried online at http://camptocamp.github.io/puppet-catalog-diff-viewer.
+The interface can be tried online at http://voxpupuli.github.io/puppet-catalog-diff-viewer.
```

Comment:
We redirect to https://voxpupuli.org/puppet-catalog-diff-viewer/, please directly link to that.

Reply:
Good point directly linked